docs/support.md

```diff
@@ -0,0 +1,33 @@
+# Customer Support
+
+In context of CDN360 we consider 3 types of customers.
+
+## Customer Types
+
+1. **Trial Customer** - Customers who onboarded through the Web UI, but have not upgraded to paid customers. These customers have a Trial credit of $50. When they reach $50 consumption, their properties may be undeployed from production automatically. To continue working with the services, an upgrade to Paid Customer is required. No credit card is required to sign-up and start using the Trial account. 
+
+    One important thing to add is that *Trial Accounts* have limitations. By default it is impossible to deploy to a production environment. We suggest setting up a property, deploying it to the staging environment and testing if it behaves properly. After that it is necessary to contact customer support to enable deployment to production.
+
+
+2. **Paid Customer** - Customers who upgraded through the Web UI using their credit card. These customers are also referred to as standard or month-to-month paid customers. They can stop using service at any time. Paid customers are obliged to pay a minimum amount of $50 per month even if the service consumption is lower than that.
```

Comment:
stop using "the" service ?

Reply:
I would reword as follows:

"*Trial Accounts* have limitations that prevent properties from being deployed to a production environment. We suggest that users with Trial Accounts set up a property, deploy it to the staging environment, and then test the property to determine whether it behaves as expected. If it does, contact customer support to allow deployment to production.


2. **Paid customer** - Customers who upgraded through the Web UI using their credit card. These customers are also referred to as standard or month-to-month paid customers. They can stop using the service at any time. Paid customers are obligated to pay a minimum of $50 per month, even if the service consumption fee is lower than this minimum cost."